main.go

```diff
@@ -0,0 +1,121 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+)
+
+// TODO: add language flag diversion, unit unit test
+
+func main() {
+	// peeper subcommand "get"
+	getCmd := flag.NewFlagSet("get", flag.ExitOnError)
+	// Flags for "get" subcommand
+	// ("flag name", default, "Flag explanation")
+	getAll := getCmd.Bool("all", false, "Get full shortcut list")
+	getByKeyword := getCmd.String("keyword", "", "Find a shortcut with keyword ")
+
+	// peeper subcommand "add"
+	addCmd := flag.NewFlagSet("add", flag.ExitOnError)
+	// Flags(inputs) for "add" subcommand
+	addName := addCmd.String("name", "", "Name of the shortcut")
+	addShortcut := addCmd.String("shortcut", "", "Shortcut")
```

Comment:
Presumably both of these are required right? In that case, perhaps they should be arguments (instead of flags)

Reply:
>both of these are required
Yes as {"Quite the app", "Cmd + Q"}

> perhaps they should be arguments
Thank you, will find how to do this.